version.inc.php

```diff
@@ -6,7 +6,7 @@
 
 function GetCodeVersion()
 {
-	return "v3.4.2.646";
+	return "v3.4.2.647";

```

Comment:
Priste prosim povysit i sub verzi. 3.4.3 napr :)
Mergnu to a povysim to ted sam.